poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
Poetry needs to be updated to 1.8.4, otherwise this regresses backwards. The local command is `poetry self update`.  From there, you can run `poetry update utils` and this file should reset it back to 1.8.4.

Reply:
Oh, nice catch, thanks!